jobs/statistics/CompileCounterSubmissionDailyMetrics.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * @file jobs/statistics/CompileCounterSubmissionDailyMetrics.php
+ *
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class CompileCounterSubmissionDailyMetrics
+ *
+ * @ingroup jobs
+ *
+ * @brief Compile COUNTER submission daily metrics.
+ */
+
+namespace APP\jobs\statistics;
+
+use APP\statistics\TemporaryItemInvestigationsDAO;
+use APP\statistics\TemporaryItemRequestsDAO;
+use APP\statistics\TemporaryTotalsDAO;
+use PKP\db\DAORegistry;
+use PKP\jobs\BaseJob;
+
+class CompileCounterSubmissionDailyMetrics extends BaseJob
+{
+    /**
+     * The number of times the job may be attempted.
+     */
+    public $tries = 1;
+
+    /**
+     * The load ID = usage stats log file name
+     */
+    protected string $loadId;
+
+    /**
+     * Create a new job instance.
+     */
+    public function __construct(string $loadId)
```

Comment:
You can use this new syntax to declare the properties that should be also set at the constructor:
```php
public function __construct(protected string $loadId) {
}
```